subnet/subnet.go

```diff
@@ -25,7 +25,8 @@ import (
 )
 
 type LeaseAttrs struct {
-	PublicIP    ip.IP4
+	InterfaceIP ip.IP4
```

Comment:
You will need to remove all of its usage as well. Just go back to using `PublicIP`. The Interface IP/device is only needed in those places where you're binding to the interface. In fact the only place it is needed is in `vxlan` backend. It should probably also be used in `udp` backend to explicitly bind to the specified interface/IP. However that can wait.